ccd-definition/CaseField.json

```diff
@@ -130,6 +130,30 @@
     "FieldType": "Label",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "courtBundleLabel",
+    "Label": "# 5. Court bundle",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "courtBundleLabelEvidence",
+    "Label": "## 5. Court bundle",
```

Comment:
Do we need to have 2 instances of label? Seems stupid to have it only to achieve slightly different font size.

Please talk to design so that we can avoid that.

Reply:
problem comes is that the tab showing documents looks silly. The others are all complex types so have the grey banner. Court bundle does not have this so goes in there unlabelled.